Plugins/DeleteUnusedBranches/DataGridViewCheckBoxHeaderCell.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Drawing;
+using System.Windows.Forms;
+using System.Windows.Forms.VisualStyles;
+
+namespace DeleteUnusedBranches
+{
+    public delegate void CheckBoxClickedHandler(bool state);
```

Comment:
event handlers must follow https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/event

Reply:
Thanks for the reference, I learned something new. Done.